src/painter.rs

```diff
@@ -92,7 +92,7 @@ impl Painter {
                 Some(pos) => {
                     let old_image = self.paints.remove(id).unwrap().image;
 
-                    let mut surface = Surface::new_raster_n32_premul(skia_safe::ISize::new(
+                    let mut surface = surfaces::raster_n32_premul(skia_safe::ISize::new(
```

Comment:
Since it seems like the surface api changed, you should probably also bump the minimum version of skia I think?